configs/templates/ufs-srw-public-v2/spack.yaml

```diff
@@ -19,7 +19,7 @@ spack:
 
   specs:
   - jasper@2.0.25
-  - zlib@1.2.12
+  - zlib@1.2.13
```

Comment:
i believe that the package version numbers for ufs application release branches (e.g.: this spack.yaml for ufs-srw release public v2) should be static, as the module versions required by those applications are fixed (e.g.: https://github.com/ufs-community/ufs-srweather-app/blob/release/public-v2/modulefiles/srw_common.spack). 

that is just to say, we need zlib to be 1.2.11 (it should not have been updated to 1.2.12 previously, either) for ufs-srw-public-v2. ufs-srw-dev's spack.yaml should be continually updated, though.

Reply:
I can revert this, no problem. I may have to add `deprecated: true` to the spack config section above. Will do after the CI tests finished, because there is no need to abort those (`ufs-srw-v2` is not CI tests, `ufs-srw-dev` is).

Reply:
@climbfuji sounds good and thank you! i didn't even catch the zlib issue w/ this spack.yaml earlier. we have a deprecated: true statement in the srw-dev spack.yaml for the same reason.

Reply:
I made this change in my latest commit (https://github.com/NOAA-EMC/spack-stack/pull/390/commits/dbf89629a15a661152761abee273d5f6935aa3e6), merging now.